.travis.yml

```diff
@@ -27,7 +26,6 @@ deploy:
   on:
     tags: true
     all_branches: true
-    rvm: 2.0
```

Comment:
Thank you for removing 2.0! That does mean we need a little more in the CHANGLOG. Could you add the following, also under `Unreleased`: 

```
### Breaking Changes
Dropping ruby `< 2.1` support (@yuri-zubov)
```

Reply:
technically this is just removing the testing, not removing support, the `.gemspec` needs to be updated so I will do that prior to release.